src/views/DataView/index.tsx

```diff
@@ -313,7 +313,8 @@ const DataView = ({
     <div className={
         `data-view ${detailPanelIsOpen
           ? 'data-view--squished'
-          : ''}`}
+          : ''}`
+}
```

Comment:
looks like something might be off with the spacing of the autofix here

Reply:
fixed in [3ae605b](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/3ae605b4002a665a6ea1e848c37f3bb098e6fec5)